app/services/load_events_from_github_service.rb

```diff
@@ -39,7 +38,28 @@ def add_new_event?(event)
     event.event == LABELED_EVENT && Event.find_by(github_id: event.id).nil?
   end
 
+  def all_issues_for_needed_period
+    all_owners_and_repos_array.reduce([]) do |array, owner_repo_array|
+      array << github.issues.list(
+        user: owner_repo_array.first,
+        repo: owner_repo_array.last,
+        since: since_issues_value,
+        per_page: EVENTS_PER_PAGE
+      ).reduce([]) do |arr, issue|
+        arr << { user: owner_repo_array.first, repo: owner_repo_array.last, number: issue.number }
+      end
+    end.flatten.compact
+  end
+
+  def since_issues_value
+    Event.any? ? last_event_formatted_created_date : DEFAULT_SINCE_VALUE
```

Comment:
Look like we should link events to user account. In case of other user will log in we will have a lot of errors.